projects/samples/devices/controllers/sample_supervisor/sample_supervisor.c

```diff
@@ -29,6 +29,12 @@ int main(int argc, char *argv[]) {
 
   root = wb_supervisor_node_get_root();
   children = wb_supervisor_node_get_field(root, "children");
+  void wb_supervisor_field_import_mf_node(children, -1, "MyBot.wbo");
+  printf("Imported a node at the last position of the scene tree.")
```

Comment:
```suggestion
  printf("Imported a node at the last position of the scene tree.");
```

Reply:
Hi, the changes I made in [https://github.com/cyberbotics/webots/pull/1691/commits/b4be8c01dc3e0fc493c392e36cdd780f0d43b49e](url) are failing. Could you indicate the mistakes?
@omichel 
@DavidMansolino

Reply:
Here is the diff (it is mainly because of missing space before comments):
```diff
DIFF:*** 

--- 

***************

*** 28,40 ****

    int n, i;

  

    wb_robot_init();

! 

!   root = wb_supervisor_node_get_root();                          /*A handle to a node in the world.*/

!   children = wb_supervisor_node_get_field(root, "children");     /*Returns a handler to a node field.*/

    wb_supervisor_field_import_mf_node(children, -1, "MyBot.wbo"); /*Imports a node from a .wbo file*/

    printf("Imported a node at the last position of the scene tree.");

!   n = wb_supervisor_field_get_count(

!     children); /*Returns number of items of MF passed as argument. If SF or NULL is passed -1 is returned.*/

    printf("This world contains %d nodes:\n", n);

    wb_supervisor_field_remove_mf(children, -1); /*Removes a node as specified as an argument from the Webots scene tree.*/

    printf("Removed the last node.");

--- 28,39 ----

    int n, i;

  

    wb_robot_init();

!   

!   root = wb_supervisor_node_get_root(); /*A handle to a node in the world.*/

!   children = wb_supervisor_node_get_field(root, "children"); /*Returns a handler to a node field.*/

    wb_supervisor_field_import_mf_node(children, -1, "MyBot.wbo"); /*Imports a node from a .wbo file*/

    printf("Imported a node at the last position of the scene tree.");

!   n = wb_supervisor_field_get_count(children); /*Returns number of items of MF passed as argument. If SF or NULL is passed -1 is returned.*/

    printf("This world contains %d nodes:\n", n);

    wb_supervisor_field_remove_mf(children, -1); /*Removes a node as specified as an argument from the Webots scene tree.*/

    printf("Removed the last node.");

***************

*** 53,59 ****

    wb_robot_step(8000);                                 /* wait for 8 seconds */

    node = wb_supervisor_field_get_mf_node(children, 3); /* PointLight */

    field = wb_supervisor_node_get_field(node, "location");

!   wb_supervisor_field_set_sf_vec3f(field, location); /*Assigns a value to a specified single field.*/

    printf("Moved location of the PointLight!\n");

    while (wb_robot_step(32) != -1) {

    }

--- 52,58 ----

    wb_robot_step(8000);                                 /* wait for 8 seconds */

    node = wb_supervisor_field_get_mf_node(children, 3); /* PointLight */

    field = wb_supervisor_node_get_field(node, "location");

!   wb_supervisor_field_set_sf_vec3f(field, location);/*Assigns a value to a specified single field.*/

    printf("Moved location of the PointLight!\n");

    while (wb_robot_step(32) != -1) {

    }

----------------------------------------------------------------------
```

Note also that we use `//` intead of `/* ... */` for commenting single lines.